### ISSUE TITLE
Trezor Safe 3 not supported

### ISSUE THREAD

**Issue by TrbnH** [cryptoadvance/specter-desktop#2414]
Adding a Trezor Safe 3 does not work when selecting "Trezor" as device.

1. Go to 'Add Device'
2. Click on 'Trezor'
3. Click on 'Get via USB'
4. See error 'Internal error: unsupported trezor model'

 - Windows desktop
 - Specter Version [2.0.2]

**Comment by moneymanolis** [comment]
Might be as easy as updating the hwi version to the one that supports Trezor 3.

**Comment by moneymanolis** [comment]
This is the relevant PR on the hwi side: https://github.com/bitcoin-core/HWI/pull/716

**Comment by PeteClubSeven** [comment]
> Might be as easy as updating the hwi version to the one that supports Trezor 3.

I'll try to test this on the weekend 🥳

**Comment by bitcoinlizard** [comment]
I don't believe this will work.  I've found that Specter Desktop doesn't work with HWI versions 2.3 and newer:

https://github.com/cryptoadvance/specter-desktop/pull/2299#issuecomment-1734147837

**Comment by PeteClubSeven** [comment]
> I don't believe this will work. I've found that Specter Desktop doesn't work with HWI versions 2.3 and newer:
> 
> [#2299 (comment)](https://github.com/cryptoadvance/specter-desktop/pull/2299#issuecomment-1734147837)

Yes you're right, I tried upgrading HWI to 2.4.0 and encountered a similar error
